### ISSUE TITLE
Space after <c> tag is removed

### ISSUE THREAD

**Issue by SIkebe** [OmniSharp/omnisharp-roslyn#1046]
In xml documents, a space after `<c>` tag is removed.
`<c>DoWork</c> is a method in the <c>TestClass</c> class.` returns `DoWorkis a method in the TestClassclass.`.

![xml-doc-c-tag](https://user-images.githubusercontent.com/17608272/33692338-2ce55102-db30-11e7-8836-048eb176be41.png)

Here's the omnisharp logs.

```
        ************ Request ************
{"Type":"request","Seq":714,"Command":"/typelookup","Arguments":{"FileName":"<file path>","Line":83,"Column":35,"IncludeDocumentation":true}}
[dbug]: OmniSharp.Stdio.Host
        ************  Response ************ 
{"Request_seq":714,"Command":"/typelookup","Running":true,"Success":true,"Message":null,"Body":{"Type":"void TestClass.DoWork(int Int1)","Documentation":"DoWorkis a method in the TestClassclass.\n"},"Seq":2541,"Type":"response"}
```

**Comment by rchande** [comment]
@akshita31 Can you take a look?

**Comment by akshita31** [comment]
This issue has been fixed in the 1.14 milestone. We just released a beta of 1.14 - https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.14.0-beta4. Follow [these instructions](https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.14.0-beta4) to install a beta release of C# for VS Code.
